portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -98,11 +108,27 @@ private static String getParameter(HttpServletRequest request, String name, Stri
     return value;
   }
 
-  private static void storeComments(String name, String email, String comment, long timestamp) {
+  private static Optional<Double> getSentiment(String message) {
+    try {
+      Document doc =
+          Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+      LanguageServiceClient languageService = LanguageServiceClient.create();
+      Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+      double score = sentiment.getScore();
+      languageService.close();
+      return Optional.of(score);
+    } catch (IOException e) {
+      System.err.println(e.getMessage());
```

Comment:
change e.getMessage() to a different error message when catching IOException - e.getMessage will return null https://stackoverflow.com/questions/40979858/ioexception-returning-null-for-get-message

Reply:
Done